tests/layer_tests/tensorflow_tests/test_tf_Pack.py

```diff
@@ -33,9 +31,7 @@ def create_pack_net(self, input_shape, input_num, axis, input_type):
             else:
                 tf.raw_ops.Pack(values=inputs)
             tf.compat.v1.global_variables_initializer()
-
             tf_net = sess.graph_def
-
```

Comment:
please revert changes with deleted blank lines above. They are needed for readability.

Reply:
I will do that. Thank you